cocos/core/gfx/webgl2/webgl2-commands.ts

```diff
@@ -1314,17 +1338,17 @@ export function WebGL2CmdFuncCreateFramebuffer (device: WebGL2Device, gpuFramebu
             gl.bindFramebuffer(gl.FRAMEBUFFER, gpuFramebuffer.glFramebuffer);
         }
 
-        for (let i = 0; i < gpuFramebuffer.gpuColorTextures.length; ++i) {
-            const colorTexture = gpuFramebuffer.gpuColorTextures[i];
+        for (let i = 0; i < gpuFramebuffer.gpuColorViews.length; ++i) {
+            const colorTexture = gpuFramebuffer.gpuColorViews[i].gpuTexture;
```

Comment:
the framebuffer size is not calculated here, so begin render pass might use the wrong viewport size?